src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '0.133.5'
+__version__ = '0.134.0'
```

Comment:
should this be 1.0.0?

Reply:
Not yet. We've got a few more small things to do, but they're not breaking. So this is like a "soft release": if a user gets their code up to date with this version, then they are ready for v1.